storage/transfermanager/downloader.go

```diff
@@ -452,16 +452,34 @@ type DownloadRange struct {
 
 // DownloadObjectInput is the input for a single object to download.
 type DownloadObjectInput struct {
-	// Required fields
-	Bucket      string
-	Object      string
+	// Bucket is the bucket in GCS to download from. Required.
+	Bucket string
+
+	// Object is the object in GCS to download. Required.
+	Object string
+
+	// Destination is the WriterAt to which write the bytes, such as an [os.File]
```

Comment:
...to which the Downloader will write the bytes? Seems like a word is missing. Also I would say "object data" over "bytes".

Reply:
Thanks, that's better